noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -458,16 +458,19 @@ contract Test {
         constant.value
     }
 
+    unconstrained fn store_in_pxe_store(key: Field, values: [Field; 3]) {
+        store(key, values);
```

Comment:
I'd rather import `pxe_store` and then do `pxe_store::load` and `pxe_store::store`, which seems a bit more readable. `pxe_store::store` makes me think that we perhaps want `pxe_db::store` instead.

Reply:
Good idea. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/10867/commits/4821ed0489f2f356aea21c8f7f6d82f2c13ed078